packages/airnode-examples/integrations/relay-security-schemes/create-config.ts

```diff
@@ -0,0 +1,150 @@
+      type: 'evm',
+      options: {
+        txType: 'eip1559',
+        baseFeeMultiplier: '2',
+        priorityFee: {
+          value: '3.12',
+          unit: 'gwei',
+        },
+      },
+    },
+  ],
+  nodeSettings: {
+    cloudProvider: createCloudProviderConfiguration(generateExampleFile),
+    airnodeWalletMnemonic: '${AIRNODE_WALLET_MNEMONIC}',
+    heartbeat: {
+      enabled: false,
+    },
+    httpGateway: {
+      enabled: false,
+    },
+    logFormat: 'plain',
+    logLevel: 'INFO',
+    nodeVersion: createNodeVersion(),
+    stage: 'dev',
+    skipValidation: true,
+  },
+  triggers: {
+    rrp: [
+      {
+        endpointId: '0x825a9b8e4e78772cd07cc4307de6737b67cf6d624fff2fa56f71318e479f624b',
+        oisTitle: 'Relay Security Schemes via httpbin',
+        endpointName: 'httpbinRelaySecuritySchemes',
+      },
+    ],
+  },
+  ois: [
+    {
+      oisFormat: '1.0.0',
+      title: 'Relay Security Schemes via httpbin',
+      version: '1.0.0',
+      apiSpecifications: {
+        servers: [
+          {
+            url: 'https://httpbin.org',
+          },
+        ],
+        paths: {
+          '/get': {
+            get: {
+              parameters: [],
+            },
+          },
+        },
+        components: {
+          securitySchemes: {
+            relayRequesterAddress: {
+              in: 'header',
+              type: 'relayRequesterAddress',
+              name: 'requesterAddress',
+            },
+            relaySponsorAddress: {
+              in: 'query',
+              type: 'relaySponsorAddress',
+              name: 'sponsorAddress',
+            },
+            relaySponsorWalletAddress: {
+              in: 'query',
+              type: 'relaySponsorWalletAddress',
+              name: 'sponsorWalletAddress',
+            },
+            relayChainId: {
+              in: 'query',
+              type: 'relayChainId',
+              name: 'chainId',
+            },
+            relayChainType: {
+              in: 'query',
+              type: 'relayChainType',
+              name: 'chainType',
+            },
+          },
+        },
+        security: {
+          relayChainId: [],
+          relayChainType: [],
+          relayRequesterAddress: [],
+          relaySponsorAddress: [],
+          relaySponsorWalletAddress: [],
+        },
+      },
+      endpoints: [
+        {
+          name: 'httpbinRelaySecuritySchemes',
+          operation: {
+            method: 'get',
+            path: '/get',
+          },
+          fixedOperationParameters: [],
+          reservedParameters: [
+            {
+              name: '_type',
+              fixed: 'address,address,address,uint256,string32',
+            },
+            {
+              name: '_path',
+              fixed:
+                'headers.Requesteraddress,args.sponsorAddress,args.sponsorWalletAddress,args.chainId,args.chainType',
+            },
+            {
+              name: '_times',
+              fixed: '',
```

Comment:
Mhh, does this work? I think it should error out and the correct value should be `,,,,`

Reply:
Yea that actually does work. I've done some more testing:
Works in `config.json`:
- `"fixed": ""`
- `"fixed": ",,,,"`
- `"fixed": ",,,10,"`  (since the 4th parameter is chainId, this returns `313370`)

Does not work (more to prove I'm testing correctly):
- `"fixed": ",,,,999"`  (since the 5th parameter is not an int type)

I've gone ahead and updated it to `,,,,` anyway as a precaution

Reply:
Mhh, it seems this doesn't work then https://github.com/api3dao/airnode/blob/master/packages/airnode-adapter/src/response-processing/extraction.ts#L87

can you investigate? I find it strange, since there are even tests for it https://github.com/api3dao/airnode/blob/master/packages/airnode-adapter/src/response-processing/extraction.test.ts#L197

Reply:
I'll merge this and investigate separately as I've already updated it to `,,,,` as a precaution, but interestingly none of the tests in the block you linked to include `_times`.

Reply:
> I'll merge this and investigate separately

Makes sense.

> but interestingly none of the tests in the block you linked to include _times.

Yeah, but the logic is that it checks the number of values in `_type` and enforces _path and _times values to have same length - which seems correct to me.